deployment/armada-bundle/values.yaml

```diff
@@ -25,40 +25,10 @@ postgresql:
   auth:
     postgresPassword: psw
 
-armada-lookout-migration:
-  image:
-    tag: v0.3.29
-  ingressClass: nginx
-  clusterIssuer: dev-ca
-
-  prometheus:
-    enabled: true
-
-  applicationConfig:
-    nats:
-      servers:
-        - nats://nats.default.svc.cluster.local:4222
-      clusterId: nats
-      subject: ArmadaTest
-      queueGroup: ArmadaLookoutEventProcessor
-    postgres:
-      maxOpenConns: 100
-      maxIdleConns: 25
-      connMaxLifetime: 30m
-      connection:
-        host: postgresql.default.svc.cluster.local
-        port: 5432
-        user: postgres
-        password: psw
-        dbname: postgres
-    uiConfig:
-      armadaApiBaseUrl: "http://armada.default.svc.cluster.local:8080"
-    prunerConfig:
-      daysToKeep: 42
-      batchSize: 1000
-
 stan:
-  nameOverride: "nats"
+  stan:
```

Comment:
We should be able to drop stan now. Since we're populating lookout from Pulsar I don't think we have anything left using stan/nats. Also, have a look at the most recent version of the quickstart to see what they're doing there.

Reply:
On Friday we had tons of issues due to STAN and cluster id. Can Armada Server and Lookout work with Pulsar only now (without STAN as streaming backend)?

Reply:
Yes, it can. Set `disableEventProcessing` (I think that's the right name) to `true` in lookout and make sure to start the `lookout-ingester`. I think the quickstart does this now.